### ISSUE TITLE
Unable to log in through standard Devise routes after adding gem

### ISSUE THREAD

**Issue by waffleau** [waiting-for-dev/devise-jwt#61]
I've added `devise-jwt` to an app that was just using regular Devise form logins, with the intention of allowing normal logins alongside API access.

After adding the gem, I can authenticate and retrieve a JWT token as expected, but I'm not longer to log in via the HTML form, receiving the error "You need to sign in or sign up before continuing."

Exert from the Rails console: 

```
Started POST "/users/sign_in" for 127.0.0.1 at 2017-12-29 08:16:23 +1100
   (1.4ms)  SELECT "schema_migrations"."version" FROM "schema_migrations" ORDER BY "schema_migrations"."version" ASC
Processing by Devise::SessionsController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"u6kRGThVq/n/iam4z6PRug1AGsmSKhV7FV/sK4Q+Rh10JWPrfLxdn80wfzD96IBX2TcnOV+K/CpK4z1vbux8Qg==", "user"=>{"email"=>"test@test.com", "password"=>"[FILTERED]", "remember_me"=>"0"}, "commit"=>"Sign in"}
  User Load (2.4ms)  SELECT  "users".* FROM "users" WHERE "users"."email" = $1 ORDER BY "users"."id" ASC LIMIT $2  [["email", "test@test.com"], ["LIMIT", 1]]
   (0.4ms)  BEGIN
  SQL (1.2ms)  UPDATE "users" SET "current_sign_in_at" = $1, "last_sign_in_at" = $2, "sign_in_count" = $3, "updated_at" = $4 WHERE "users"."id" = $5  [["current_sign_in_at", "2017-12-28 21:16:24.057664"], ["last_sign_in_at", "2017-12-28 21:11:00.022634"], ["sign_in_count", 89], ["updated_at", "2017-12-28 21:16:24.060092"], ["id", 1]]
   (6.3ms)  COMMIT
Redirected to http://localhost:3000/dashboard
Completed 302 Found in 291ms (ActiveRecord: 23.4ms)


Started GET "/dashboard" for 127.0.0.1 at 2017-12-29 08:16:24 +1100
Processing by DashboardController#index as HTML
  User Load (0.8ms)  SELECT  "users".* FROM "users" WHERE "users"."id" = $1 ORDER BY "users"."id" ASC LIMIT $2  [["id", 1], ["LIMIT", 1]]
   (0.4ms)  BEGIN
   (0.3ms)  COMMIT
Completed 401 Unauthorized in 10ms (ActiveRecord: 1.6ms)
```

And my config for the gem:

```
config.jwt do |jwt|
  jwt.secret = Rails.application.secrets.jwt_secret_key

  jwt.dispatch_requests = [
    ['POST', %r{^/api/v1/access(.json)?$}],
    ['POST', %r{^/users/sign_in$}]
  ]
end
```

I'm using the JTI revocation strategy, and my feeling is that when logging in through regular old Devise an auth check using JTI is failing. I only care about JWT auth when using `/api/*` routes, so if there's a way to disable JWT/JTI checks for other routes that would be a suitable solution.

Let me know if there's any other info I can provide.

**Comment by waiting-for-dev** [comment]
Can you test with the `devise-jwt-61` branch of the `warden-jwt_auth` gem?

So, in your Gemfile, alongside the `gem 'devise-jwt'`:

```ruby
gem 'warden-jwt_auth', github: 'waiting_for_dev/warden-jwt_auth', branch: 'devise-jwt-61'
```

**Comment by waffleau** [comment]
It appears to be working using the branch. :) Is the recommended solution for now to just stick with the branch, or is this a fix branch?

**Comment by waiting-for-dev** [comment]
I created that branch to confirm my hypothesis... I'll fix it in master as soon as possible

**Comment by waffleau** [comment]
Great, thanks for the quick work!

**Comment by waiting-for-dev** [comment]
Hey @waffleau , could you please update your application to contain latest commits in `devise-jwt-61` branch and report everything is still working for both API and HTML authentication?

**Comment by waffleau** [comment]
I've just updated and all appears to working still (commit `01bf47c`).

**Comment by waiting-for-dev** [comment]
Great. Fixed in master and released in version `0.5.3`. Thanks for reporting!

**Comment by waffleau** [comment]
Excellent, thanks for your help!
